staging/src/k8s.io/component-base/metrics/testutil/metrics.go

```diff
@@ -281,21 +281,6 @@ func (hist *Histogram) Average() float64 {
 	return hist.GetSampleSum() / float64(hist.GetSampleCount())
 }
 
-// Clear clears all fields of the wrapped histogram
-func (hist *Histogram) Clear() {
```

Comment:
Is this method only called by scheduler?

Reply:
As far as I can tell, yes.